### ISSUE TITLE
Can not resume socket accept process: The parameter is incorrect. (os error 87)

### ISSUE THREAD

**Issue by danylaporte** [actix/actix-web#905]
**[actix_server::accept] Can not resume socket accept process: The parameter is incorrect. (os error 87)**

I have this message from time to time. I don't know how to reproduce it. It was working correctly with actix-web 0.73 but occurs with 1.0, I am on Windows 10.

When I have this message in the log, actix-web won't serve any request.

Any idea?

**Comment by damalmax** [comment]
This is probably a bug in `actix-web:1.0.0`. I also tried to update my application to use the new version and I've found that it stops to accept new connections under the load with JMeter (both Windows/Linux envs).

I've verified that the is reproducible with simple application from `Readme.md`:
```rust
use actix_web::{web, App, HttpServer, Responder};

fn index(info: web::Path<(u32, String)>) -> impl Responder {
    format!("Hello {}! id:{}", info.1, info.0)
}

fn main() -> std::io::Result<()> {
    HttpServer::new(
        || App::new().service(
            web::resource("/{id}/{name}/index.html").to(index)))
        .bind("127.0.0.1:4141")?
        .run()
}
```

Cargo.toml:
```toml
[package]
name = "actix-web-example"
version = "0.1.0"
edition = "2018"

[dependencies]
actix-web = "1.0.0"
```

Jmeter config:
- Number of threads: 5
- Ramp-Up Period (in seconds): 5
- Loop count: 10000
HTTP Request sampler:
GET http://localhost:4141/{id}/{name}/index.html

The application doesn't accept new requests after 10-30 seconds of the test (result is random). Also, I don't see any error messages in the log. Actually, after some time (load test is stopped), it starts to handle new requests.

Failed Jmeter requests have the following response body:
```
java.net.BindException: Address already in use: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
```

**Comment by danylaporte** [comment]
This issue seems to appear when a request is made during the initialization phase of actix-web.

**Comment by ciyool** [comment]
The same error occurred in my application.
 who know the cause and how to fix it?

**Comment by JohnTitor** [comment]
@danylaporte @ciyool I just released actix-server v1.0.2. Can you check if it resolves the issue (again)?

**Comment by danylaporte** [comment]
@JohnTitor Seems to be resolved. Thank you!
